tests/test_integration.py

```diff
@@ -643,6 +682,21 @@ def test_rollback_drbd(self):
 
         self.check_vm_present()
 
+    def test_rollback_xfs(self):
+        obj = Query({'hostname': VM_HOSTNAME}, ['puppet_environment']).get()
```

Comment:
You should be able to access it via `self.vm_obj` without doing an extra Query.

Reply:
True, but it's done in the same way for every rollback test. Should I change it for everything else too?

Reply:
If every other test queries a new object, maybe it is necessary for something? I vaguely remember there was some magic happening here.

Reply:
I'd like to make it consistent with other tests for now. If you want, I could leave a TODO comment for future

Reply:
Fair enough. Lets just make a todo.

Reply:
TODO added as a dedicated commit